src/util.js

```diff
@@ -18,4 +18,18 @@ function createSCM(item) {
     default:
       return new Github(item.baseUrl, item.user, item.token);
   }
+}
+
+function getGitHubJSON(url, accessToken, data) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/15f4157afeba0560f3f12968d583a6f4fceda1cf/src/util.js#L23"></span>‘getGitHubJSON’ is defined but never used.</td>
    </tr>
  </tbody>
</table>